department-of-veterans-affairs/caseflow
# Move test veterans list to separate page

Connects #12934 

### Description
This PR breaks out the list of local veteran records to a new page located at `/test/data`, making the `/test/users` page much faster to use, for the more common task of switching users.

### Acceptance Criteria
- [x] Make a `/test/data` page and move the list of veterans to that page
  - Note: engineers choice whether to migrate to another Rails view or handle via front-end routing
- [x] Link to the test data page from the test users page
- [ ] Check in with Training, announce in appeals-engineering

### Testing Plan
1. `/test/users`
2. Bask in how fast it loads
3. Check that the link near the bottom goes to `/test/data`
4. Check that `/test/data` displays local veteran records

### User Facing Changes
Link from `/test/users`:
<img width="1104" src="https://user-images.githubusercontent.com/282869/89545690-adaaa580-d7d1-11ea-9eb7-fd6736b84501.png">

Inaugural content on `/test/data`:
<img width="1111" src="https://user-images.githubusercontent.com/282869/89545844-db8fea00-d7d1-11ea-97a9-81bd912ac9f9.png">
